installing/installing_with_agent_based_installer/installation-config-parameters-agent.adoc

```diff
@@ -0,0 +1,19 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installation-config-parameters-agent"]
+= Installation configuration parameters for the Agent-based Installer
+include::_attributes/common-attributes.adoc[]
+:context: installation-config-parameters-agent
+
+toc::[]
+
+Before you deploy an {product-title} cluster using the Agent-based Installer, you provide parameters to customize your cluster and the platform that hosts it.
+When you create the `install-config.yaml` and `agent-config.yaml` files, you must provide values for the required parameters, and you can use the optional parameters to customize your cluster further.
+
+include::modules/installation-configuration-parameters.adoc[leveloffset=+1]
+
+include::modules/agent-configuration-parameters.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../../installing/installing_with_agent_based_installer/prepare-pxe-assets-agent.adoc#prepare-pxe-assets-agent[Preparing PXE assets for {product-title}].
+* xref:../../installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc#root-device-hints_ipi-install-installation-workflow[Root device hints].
```

Comment:
```suggestion
* xref:../../installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc#root-device-hints_ipi-install-installation-workflow[Root device hints]
```

Reply:
This is a duplicate of the comment below, so I'm assuming this is meant for the first list item in this Additional Resources list. I'll remove periods from both items.

Reply:
ooops. yes :)